tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -24,6 +24,7 @@ class Contacts(Base):
     _no_contacts_message_locator = (By.CSS_SELECTOR, '*[data-l10n-id="no-contacts"]')
     _group_container_selector = "#groups-container"
     _contact_locator = (By.CSS_SELECTOR, 'li[data-uuid]:not([data-group="ice"])')
+    _screen_locator = (By.ID, 'view-contacts-list')
```

Comment:
actually on second thought, I changed the name on mine to _page_locator, because it's technically referring to a page view